### ISSUE TITLE
Entities resolving doesn't work without explict vdoc

### ISSUE THREAD

**Issue by Koc** [Haehnchen/idea-php-symfony2-plugin#287]
``` php
$em = $this->getDoctrine()->getManager();
/* @var $em EntityManager */
$payment = $em->find('MetalServicesBundle:Payment', $result->getInvId());

$payment-> // autosuggest doesn't work here when I'm removing vdoc for $em valiable
// but autosuggest for entity name works
```

**Comment by Koc** [comment]
Can anybody confirm that https://youtrack.jetbrains.com/issue/WI-23727 was already fixed?

**Comment by Haehnchen** [comment]
"Not reproducible with Symfony 2.7 and PhpStorm 141.1524.", so for testing u need upcoming phpstorm eap

**Comment by MaXal** [comment]
Please try today Public Preview: https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+Early+Access+Program

**Comment by Koc** [comment]
@MaXal @Haehnchen I can confirm that issue was only **partly** fixed. As you see from mt screenshot PhpStorm highlight methods of class `Payment` like non-existent. But ctrl+click works and after ctrl+click methods aren't more highlighted.

![phpstorm-issue](https://cloud.githubusercontent.com/assets/191082/8125269/089284a6-10ed-11e5-8589-b638df2aba9c.png)

**Comment by vtexier** [comment]
![capture du 2017-02-24 11 30 25](https://cloud.githubusercontent.com/assets/2174016/23300301/4318acc6-fa85-11e6-93b6-ec17c01f1c4d.png)

I have a similar problem.

Impossible to get autocomplete for custom methods behind doctrine getRepository().

The plugin (or phpstorm) is confused by the repository class declaration.
It propose three implementations.

phpstorm 2016.3.2
symfony plugin 0.13.136
Ubuntu 14.04 64bits.

**Comment by Koc** [comment]
@Haehnchen it partly fixed in the last build (0.14.143) but not fully. Controller

```php
        $em = $this->getDoctrine()->getManager();

        $payment = $em->find('MetalServicesBundle:Payment', $payResult->getInvId());
        $payment->getCompany(); // type of the $payment is resolved
```
```php
        $em = $this->get('doctrine.orm.default_entity_manager'); // ctrl+click on service goes to EntityManager - ok

        $payment = $em->find('MetalServicesBundle:Payment', $payResult->getInvId());
        $payment->getCompany(); // type of the $payment variable not resolved
```

I've tryed to clear index after plugin installation

**Comment by Haehnchen** [comment]
@Koc what are the class instances behind `$em` `get_class($em)` in the examples?

**Comment by Koc** [comment]
Maybe it caused by #926.

**Comment by Koc** [comment]
fixed with #926 . @vtexier can you recheck last build? If doesn't work on your config - please comment

**Comment by vtexier** [comment]
After update to 2017.1.3 and last build 0.14.144, it is fixed ! Great job !
